react/src/components/ResourceAllocationFormItems.tsx

```diff
@@ -840,105 +833,274 @@ const ResourceAllocationFormItems: React.FC<
-                                      throw t(
-                                        'resourcePreset.SHMEMShouldBeSmallerThanMemory',
-                                      );
-                                    }
+                        return (
+                          <Flex direction="column" align="stretch" gap="xs">
+                            <Flex direction="row" gap={'sm'}>
+                              <ConfigProvider
+                                theme={{
+                                  components: {
+                                    Slider: {
+                                      railBg: token.colorWarningBorderHover,
+                                      railHoverBg:
+                                        token.colorWarningBorderHover,
+                                      trackBg: token.colorSuccessBorderHover,
+
+                                      trackHoverBg:
+                                        token.colorSuccessBorderHover,
+                                      railSize: token.fontSize,
+                                    },
                                   },
-                                },
-                              ]}
-                            >
-                              <DynamicUnitInputNumberWithSlider
-                                // shmem max is mem max
-                                // min={resourceLimits.shmem?.min}
-                                min={resourceLimits.shmem?.min}
-                                // max={resourceLimits.mem?.max || '0g'}
-                                addonBefore={'SHM'}
-                                max={
-                                  form.getFieldValue(['resource', 'mem']) ||
-                                  '0g'
-                                }
-                                hideSlider
-                                onChange={() => {
-                                  form.setFieldValue(
-                                    'allocationPreset',
-                                    'custom',
-                                  );
                                 }}
-                              />
-                            </Form.Item>
-                          );
-                        }}
-                      </Form.Item>
-                    </Flex>
+                              >
+                                <Slider
+                                  style={{
+                                    flex: 1,
+                                    margin: 0,
+                                    cursor: 'default',
+                                    padding: 0,
+                                  }}
+                                  styles={{
+                                    handle: {
+                                      // display: isAutoShmem ? 'none' : undefined,
+                                      display: 'none',
+                                      top: 2,
```

Comment:
```suggestion
                                    handle: {
                                      display: 'none',
```

Reply:
Sorry, I missed this suggestion. I submitted. Please check again.